Frameworks/UIKit/UIView.mm

```diff
@@ -2252,12 +2251,16 @@ - (CGPoint)convertPoint:(CGPoint)pos fromView:(UIView*)fromView {
 */
 - (void)drawLayer:(CALayer*)layer inContext:(CGContextRef)context {
     UIGraphicsPushContext(context);
+    _CGContextPushBeginDraw(context);
```

Comment:
I had the same idea, but it ended up regressing XAMLCatalog (Custom text view would not update at all).

---
In reply to: [96086902](https://github.com/Microsoft/WinObjC/pull/1705#discussion_r96086902) [](ancestors = 96086902)